package.json

```diff
@@ -3,10 +3,13 @@
   "version": "0.0.1",
   "description": "Mohit ==> m0hit",
   "scripts": {
-    "start": "node server.js"
+    "start": "node server.js",
+    "test": "node_modules/mocha/bin/_mocha ./**/*spec.js"
```

Comment:
Not working in windows.
@morgsmccauley make it `node node_modules/mocha/bin/_mocha ./**/*spec.js`